contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -18,26 +16,27 @@ contract PixelDevsUkraineDonation is
 {
     using Counters for Counters.Counter;
 
-    /// TODO: Set this to the IPFS base uri before launch
-    string public baseURI = "ipfs://abcd.../";
-    uint256 public minimumMintPrice = 12 ether;
-    bool public contractState = true;
+    string public baseURI =
+        "ipfs://QmZiCUXCytbbnqCzJAujXLxeQS7MkE9TJKyYbV6LjSinN4/";
+    bool public contractActive = true;
+    mapping(string => uint256) public tiers;
     Counters.Counter private _tokenIds;
 
+    event BaseURIUpdated(string indexed oldValue, string indexed newValue);
+    event ContractStateUpdated(bool indexed oldValue, bool indexed newValue);
     event LogTokenMinted(
         address indexed minter,
         uint256 indexed tokenId,
-        string indexed donationType
+        string indexed tier
     );
-    event BaseURIUpdated(string indexed oldValue, string indexed newValue);
-    event MinimumMintPriceUpdated(
+    event TierPricingUpdated(
+        string indexed tier,
         uint256 indexed oldValue,
         uint256 indexed newValue
     );
-    event ContractStateUpdated(bool indexed oldValue, bool indexed newValue);
 
     constructor() ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
-        // console.log("PixelDevsUkraineDonation deployed by '%s'", msg.sender);
+        //
```

Comment:
Might as well remove this poor lonely lonely comment.

Reply:
Haha the Laravel force is strong within me. Can't shake it apparently ;-) https://github.com/laravel/laravel/blob/9.x/app/Providers/AppServiceProvider.php